tekton-pipelines/buildpacks-frontend/triggers/build-pipeline-trigger.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         - taskName: kustomize
           serviceAccountName: $(tt.params.application)-build-pipeline-kustomize-sa
       pipelineRef:
-        name: frontend-buildpack-build-pipeline
+        name: buildpack-frontend-build-pipeline
```

Comment:
Let's use same names we have in our infra

```
➜ k get pipeline -n ci
NAME                                AGE
buildpack-build-pipeline            46d
dotnet-buildpack-build-pipeline     13d
frontend-buildpack-build-pipeline   30d
kaniko-build-pipeline               66d
wordpress-build-pipeline            41d
```

So we need `frontend-buildpack-build-pipeline `

Reply:
@dmitry-mightydevops that is the name from current `main` branch with your last changes in `main` for frontend buildpack, We need to actualize pipelines in `saritasa-devops-tekton` repo and `saritasa-rocks-cluster` and release a new version. By the way, do we have more projects in client infra, which uses `buildpack-frontend-build-pipeline` except mine?

Reply:
actually let's keep then as it was i.e. `buildpack-TYPE`, so the prefix is the `buildpack`, I will adjust then saritasa-rocks & cloud to use that naming so we're in sync.